jina/flow/__init__.py

```diff
@@ -795,6 +795,47 @@ def my_reader():
         """
         self._get_client(**kwargs).search(input_fn, output_fn, **kwargs)
 
+    @deprecated_alias(buffer='input_fn', callback='output_fn')
+    def eval(self, input_fn: Union[Iterator[Tuple[Union['jina_pb2.Document', bytes], Union['jina_pb2.Document', bytes]]], Callable] = None,
```

Comment:
also `eval_lines`, `eval_numpy`, ...

Reply:
could we add them as we add new evaluations and executors for specific use cases? So that we can have better testing?